src/server/api/menuBar.ts

```diff
@@ -16,6 +16,13 @@ router.post("/label", (req, res) => {
   state.activeMenuBar.tray.setTitle(label);
 });
 
+router.post("/context-menu", (req, res) => {
+  res.sendStatus(200);
+  const { contextMenu } = req.body;
+  
+  state_1.default.activeMenuBar.tray.setContextMenu(buildMenu(contextMenu));
```

Comment:
@bbredewold I think this should be simply `state.activeMenuBar` ?

Reply:
Thanks ;)